### PR TITLE
Feat/filtering ux cleanup

### PR DESCRIPTION
**Pull request by vincent-czi**
### Description

Clean up for the visual aspects of the filter panel.

#### Issue
[ch<159835>](https://app.clubhouse.io/genepi/story/<159835>)

### Test plan

Visual inspection locally so far.
Still need to put up an rdev (it's my first!) and get approval from Janeece or someone else on UI/Product.

### CODE REVIEW

**Review by phoenixAja — APPROVED** [review]
this looks good to me! Curious to see what @mmmmmmaya thinks since this was her original creation lol

File: src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx [hunk 1]
```diff
@@ -153,8 +169,39 @@ const DateFilter: FC<Props> = ({
...
+  startDate,
+  endDate,
+  deleteDateFilter,
+}: DateChipProps): JSX.Element | null {
+  // DateChip should only display if we have at least one of startDate/endDate
+  if (!startDate && !endDate) return null;
+
+  // Get the date chip message. Structure varies if only one of the two dates.
+  // Might be worth extracting date message to a common helper func elsewhere?
+  const dateIntervalLabel = `${startDate || "Prior"} to ${endDate || "today"}`;
```
Comment (phoenixAja) [inline src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx]: to reflect the designs:
```suggestion
  const dateIntervalLabel = `${startDate || "Prior"} to ${endDate || "Today"}`;
```

**Comment by mmmmmmaya** [comment]
https://app.clubhouse.io/sci-design-system/story/154464/implement-in-aspen

**Comment by vincent-czi** [comment]
@mmmmmmaya Can you give me access to the SDS clubhouse? When I click that link it gives me a little pop-up in CH telling me something about it being "Missing or you do not have access"

**Comment by phoenixAja** [comment]
in the interest of time i think we can merge this in and get design approval tomorrow? (just apply my one suggestion of capitalizing Today if you don't mind)

File: src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx [hunk 2]
```diff
@@ -79,22 +86,31 @@ const DateFilter: FC<Props> = ({
     return d.toISOString().substring(0, 10);
   };
 
+  // Use this over directly using `updateDateFilter` prop so we track filter changes.
```
Comment (mmmmmmaya) [inline src/frontend/src/components/FilterPanel/components/DateFilter/index.tsx:89]: jw, did you also test to make sure this change did not impact other places where this input is used (ie: upload page 2 for meta data)?

**Comment by vincent-czi** [comment]
That's weird: @mmmmmmaya I see the email with your comment about `DateFilter
> jw, did you also test to make sure this change did not impact other places where this input is used (ie: upload page 2 for meta data)?

But, I can't see the comment here? Did you delete the comment? Either way, I merged before seeing the email.

The answer is, crud, nope. I was just assuming `DateFilter` to be only used in the filter panel. But searching the codebase right now, I don't see it being imported outside of that area? If I'm wrong, let me know. Even if it is being used outside of the filter panel, I think it _should_ still work fine. I was trying to keep my changes purely internal to the component, so hopefully it's still fine if it's somewhere I wasn't looking while working.

**Pull request merged**